cvat-ui/src/components/workout-page/details.tsx

```diff
@@ -34,20 +35,41 @@ export default function DetailsComponent(props: DetailsComponentProps): JSX.Elem
         </div>
     );
 
+    const histogramColumns = [
+        {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+        },
+        {
+            title: 'Number of instances',
+            dataIndex: 'instances',
+            key: 'instances',
+        },
+        {
+            title: '%',
+            dataIndex: 'percent',
+            key: 'percent',
+        },
+    ];
+
+    const histogramData = (Object.keys(workout.histogram).length > 0) ? workout.histogram?.classes.map(
```

Comment:
maybe:
```javascript
const x = pred
  ? a
  : b;
```

Reply:
I've fixed this as far as I can go with used JS linter.